srcpkgs/ncurses/template

```diff
@@ -130,3 +151,22 @@ ncurses-term_package() {
 		vmove usr/share/terminfo
 	}
 }
+
+ncurses-libtinfo-libs_package() {
+	depends="ncurses-libs-${version}_${revision}"
+	short_desc+=" - libtinfo.so symlink"
+	archs=noarch
```

Comment:
I did not see this line before. Remove `archs=noarch` here and in the `-devel` subpackage.
It's technically possible, that's true, but useless and misleading, because ncurses-libs is of course architecture dependent.

Reply:
Okay, it's fixed, and I also added a short_desc to devel, which i saw was missing.